spec/01-unit/26-tracing/01-tracer_pdk_spec.lua

```diff
@@ -138,16 +178,21 @@ describe("Tracer PDK", function()
 
     it("fails set_attribute", function ()
       local span = c_tracer.start_span("meow")
-      assert.error(function() span:set_attribute("key1") end)
-      assert.error(function() span:set_attribute("key1", function() end) end)
+
+      span:set_attribute("key1")
+      assert.spy(log_spy).was_called_with(ngx.ERR, match.is_string())
+
+      span:set_attribute("key1", function() end)
+      assert.spy(log_spy).was_called_with(ngx.ERR, match.is_string())
+
       assert.error(function() span:set_attribute(123, 123) end)
     end)
 
     it("fails add_event", function ()
       local span = c_tracer.start_span("meow")
-      assert.error(function() span:set_attribute("key1") end)
-      assert.error(function() span:set_attribute("key1", function() end) end)
-      assert.error(function() span:set_attribute(123, 123) end)
+      assert.error(function() span:add_event("key1", 123) end)
+      assert.error(function() span:add_event("key1", function() end) end)
+      assert.error(function() span:add_event(123, {}) end)
```

Comment:
This seems a mistake in the original test. I'm fixing this by the way.

Reply:
i'm bit confused on why this change is needed, what was the `assert.error`previously tested upon?

Reply:
It's not needed. But the original test is not testing what it says. I'm fixing it.